docs/example.rst

```diff
@@ -1,50 +1,64 @@
-**************************
-Example: Sea state contour
-**************************
+**************************************
+Quick start example: Sea state contour
+**************************************
 
-Let's start this user guide with a simple example.
+Let’s start this user guide with a simple example, which can be copied easily line by line or downloaded from the
+examples_ section of the repository. Based on a dataset, the long-term joint distribution of sea states is estimated,
+and this distribution will be used to construct an environmental contour with a return period of 50 years. This example
+follows the OMAE2020 model by Haselsteiner et al. (2020) [1]_. The following steps are carried out in the following quick
+start example:
 
-Based on a dataset, the long-term joint distribution of sea states is estimated
-and this distribution will be used to construct an environmental contour with a
-return period of 50 years. This example follows the OMAE2020 model by
-Haselsteiner et al. (2020) ::
+1.	Load the environmental data that should be described by the joint model.
+2.	Use a predefined model to describe the dependence structure and define the parametric distributions.
+3.	Estimate the parameter values of the MultivariateModel (fitting).
+4.	Define the contour’s return period and environmental state duration.
+5.	Choose a type of contour: IFormContour, ISormContour, DirectSamplingContour or HighestDensityContour.
 
-    import numpy as np
-    import pandas as pd
-    from matplotlib import pyplot as plt
+
+Import the required virocon packages.
+::
 
     from virocon import (GlobalHierarchicalModel, get_OMAE2020_Hs_Tz,
                          calculate_alpha, IFORMContour, plot_2D_contour)
 
-    # Load sea state measurements from the NDBC buoy 46025.
+Import packages that are needed additionally.
+::
+
+    from matplotlib import pyplot as plt
 
-    data = pd.read_csv("datasets/NDBC_buoy_46025.csv", sep=",")[["Hs", "T"]]
+Load the sea state data set. Here, we use a dataset used in a benchmark study which was published at the OMAE 2019
+conference by Haselsteiner et. Al. (2019) [2]_.
+::
 
-    # Define the structure of the probabilistic model that will be fitted to the
-    # dataset. This model structure has been proposed in the paper "Global
-    # hierarchical models for wind and wave contours: Physical interpretations
-    # of the dependence functions" by Haselsteiner et al. (2020).
+    data = read_ec_benchmark_dataset("datasets/ec-benchmark_dataset_A.txt")
 
-    dist_descriptions, fit_descriptions, semantics = get_OMAE2020_Hs_Tz()
+Use a predefined model to describe the dependence structure and define the structure of the joint distribution model.
+virocon provides 4 simple predefined models, which can be directly executed. Here, we use the Hs-Tz joint distribution

```

Comment:
I think we should not mention how many predefined models we have, as this is prone to become deprecated once we add a new model. Then we would have to always update this here.